### ISSUE TITLE
The loss value is nan when I try to use resnet50 as the backbone network

### ISSUE THREAD

**Issue by zhengli97** [MingiJi/FRSKD#3]
Hi,

Thanks for your contributions!
 When I tried to implement your code on Resnet50 and Resnet101, the loss value is nan.
I just modified your code like this:
`
def cifarresnet50(pretrained=False, **kwargs):
    return CIFAR_ResNet(Bottleneck, [3, 4, 6, 3], **kwargs)
`
Then when I tried to train this network, the loss value is always nan.

(GPU: RTX 3090)

How to fix this error? Thanks!
